dll/win32/cryptui/main.c

```diff
@@ -1281,7 +1279,6 @@ static LRESULT CALLBACK cert_mgr_dlg_proc(HWND hwnd, UINT msg, WPARAM wp,
             free_certs(GetDlgItem(hwnd, IDC_MGR_CERTS));
             close_stores(GetDlgItem(hwnd, IDC_MGR_STORES));
             data = (struct CertMgrData *)GetWindowLongPtrW(hwnd, DWLP_USER);
-            ImageList_Destroy(data->imageList);
             HeapFree(GetProcessHeap(), 0, data);
             EndDialog(hwnd, IDCANCEL);
```

Comment:
This code here is intrinsically buggy (not your fault, comes from Wine).
All the destroy thing must be moved into a WM_DESTROY handler instead.

Reply:
I fully agree. Should I do it, surrounded by #ifdef __REACTOS__ ... #endif & #ifndef __REACTOS__ ... #endif ? Or do you prefer that i simply raise a WineHQ ticket and leave it as is ?

Reply:
Ideally, after checking whether these problems have been fixed in wine / wine-staging in the meantime, it would be to write the correct fix for us and send the patch to wine.

Reply:
WILCO.